packages/headless/src/controllers/folded-result-list/headless-folded-result-list.ts

```diff
@@ -66,6 +67,10 @@ export interface FoldedResultListProps {
  * The `FoldedResultList` headless controller re-organizes results into hierarchical collections (a.k.a. threads).
  */
 export interface FoldedResultList extends ResultList {
+  /**
+   * Loads all the folded results for a given collection.
+   */
+  loadAll(collection: string | Collection): void;
```

Comment:
Since this method loads all results in a collection (up to 100), I felt like this name was more exact. We could name it "show more", but I feel like that would imply that we can execute it multiple times to get more results every time.

Reply:
Good thoughts on the method name, I agree.

Why is the type a union though? Why not just: `loadAll(result: Collection): void;`
Maybe even clearer: `loadCollection(result: collection)`

Reply:
> We had an issue with the method to select facets due to it taking an object as a parameter, which is more difficult to use with vanilla JS

I'm not aware of a client whose had an issue so far. @olamothe brought up this up at some point when working in the context of Angular? I continue to believe that passing an object is the easier api design for devs working in pure js or React.

Maybe when we start working on Angular code samples I will have a different perspective, but I don't think it's good to use a union until we have a clear understanding of why.

We can always convert a single type to a union, but doing the reverse is a breaking change.